Module landing page: scrollbar styles (#26400)
mdn/content: files/en-us/web/css/css_scrollbars/index.md

---
title: CSS scrollbars styling
slug: Web/CSS/CSS_Scrollbars
page-type: css-module
spec-urls: https://w3c.github.io/csswg-drafts/css-scrollbars/
browser-compat:

>   - css.properties.scrollbar-color

Reviewer: Can we put both back in? Adding BCD entry for one can raise questions about the support for the other property.

Author: i would prefer to have neither. If we put them both in, it is huge. I'll remove them again ;)
<img width="532" alt="Screen Shot 2023-04-26 at 11 33 12 AM" src="https://user-images.githubusercontent.com/69888/234670415-ba6a22a6-7366-411a-89a7-064da079b223.png">

---

{{CSSRef}}{{SeeCompatTable}}

> The **CSS scrollbars styling** module defines properties that provide visual styling of scrollbars, including controls for the color and width of scrollbar tracks, the background of the scrollbar, and the color of scrollbar thumbs, the draggable handle of the scrollbar.

Reviewer: ```suggestion
The **CSS scrollbars styling** module defines properties that you can use for visual styling of scrollbars. You can  customize the width of the scrollbar as required. You can also customize the color of the scrollbar _track_, which is the background of the scrollbar, and the color of the scrollbar _thumb_, which is the draggable handle of the scrollbar.
```
I guess the width definition is for the scrollbar and not just for the track?

---
title: CSS scrollbars styling
slug: Web/CSS/CSS_Scrollbars
page-type: css-module
spec-urls: https://w3c.github.io/csswg-drafts/css-scrollbars/
browser-compat:
  - css.properties.scrollbar-color
  - css.properties.scrollbar-width
---

{{CSSRef}}{{SeeCompatTable}}

> The **CSS scrollbars styling** module defines properties that you can use for visual styling of scrollbars. You can  customize the width of the scrollbar as required. You can also customize the color of the scrollbar _track_, which is the background of the scrollbar, and the color of the scrollbar _thumb_, which is the draggable handle of the scrollbar.

Reviewer: ```suggestion
The **CSS scrollbars styling** module defines properties that you can use for visual styling of scrollbars. You can customize the width of the scrollbar as required. You can also customize the color of the scrollbar _track_, which is the background of the scrollbar, and the color of the scrollbar _thumb_, which is the draggable handle of the scrollbar.
```

## Reference

### CSS properties

- {{CSSxRef("scrollbar-width")}}
- {{CSSxRef("scrollbar-color")}}

## Related concepts

- {{CSSxRef("overflow-block")}} CSS property
- {{CSSxRef("overflow-inline")}} CSS property
- {{CSSxRef("overflow-x")}} CSS property
- {{CSSxRef("overflow-y")}} CSS property
- {{CSSxRef("overflow")}} CSS shorthand property
- {{CSSxRef("overflow-clip-margin")}} CSS property
- {{CSSxRef("scroll-behavior")}} CSS property
- {{CSSxRef("scrollbar-gutter")}} CSS property
- {{cssxref("scroll-margin")}} CSS shorthand property
- {{cssxref("scroll-padding")}} CSS shorthand property
- {{cssxref("scroll-snap-align")}} CSS property
- {{cssxref("scroll-snap-stop")}} CSS property
- {{cssxref("scroll-snap-type")}} CSS property
- {{CSSxRef("::-webkit-scrollbar")}} pseudo-element
- {{glossary("scroll container")}} glossary term
- {{Glossary("scrollport")}} glossary term
- ARIA [`scrollbar`](/en-US/docs/Web/Accessibility/ARIA/Roles/scrollbar_role) role

## Specifications

{{Specifications}}

## See also

> - [CSS overflow module](/en-US/docs/web/CSS/CSS_overflow)

Reviewer 2: ```suggestion
- [CSS Overflow module](/en-US/docs/Web/CSS/CSS_Overflow)
```

> - [CSS scroll snap module](/en-US/docs/web/CSS/CSS_Scroll_Snap)

Reviewer 2: Capitalization matters (see flaws)
```suggestion
- [CSS Scroll Snap module](/en-US/docs/Web/CSS/CSS_Scroll_Snap)
```

- [`@media (forced-colors: active)`](/en-US/docs/Web/CSS/@media/forced-colors) sets {{cssxref("scrollbar-color")}} to 'auto'

> - {{Experimental_Inline}} The CSS scroll timeline {{cssxref('scroll-timeline-name')}} and {{cssxref('scroll-timeline-axis')}} properties, along with the {{cssxref('scroll-timeline')}} shorthand, to create animations tied to the scroll offset of a {{glossary("scroll container")}}.

Reviewer 2: ```suggestion
- {{{cssxref('scroll-timeline-name')}}, {{cssxref('scroll-timeline-axis')}}, and {{cssxref('scroll-timeline')}} 
```

---
title: CSS scrollbars styling
slug: Web/CSS/CSS_Scrollbars
page-type: css-module
spec-urls: https://w3c.github.io/csswg-drafts/css-scrollbars/
---

{{CSSRef}}{{SeeCompatTable}}

> The **CSS scrollbars styling** module defines properties that provide visual styling of scrollbars, including controls for their color and width.

Author: ```suggestion
The **CSS scrollbars styling** module defines properties that provide visual styling of scrollbars, including controls for the color and width of scrollbar tracks, the background of the scrollbar, and the color of scrollbar thumbs, the draggable handle of the scrollbar.
```
